unlock-app/src/middlewares/lockMiddleware.js

```diff
@@ -141,6 +144,11 @@ export default function lockMiddleware({ getState, dispatch }) {
         web3Service.connect({ provider: action.provider })
       } else if (action.type === CREATE_LOCK) {
         web3Service.createLock(action.lock, getState().account)
+      } else if (action.type === UPDATE_LOCK_PRICE) {
```

Comment:
How do I actually dispatch the UPDATE_LOCK_PRICE type action from the frontend?

Reply:
This is happening through your `updateLockPrice` in your `mapDispatchToProps` function above.